_observing-your-data/ad/index.md

```diff
@@ -201,7 +205,7 @@ Be cautious when imputing extensively missing data, as excessive gaps can compro
 
 ### Suppressing anomalies with threshold-based rules
 
-In the **Advanced settings** pane, you can suppress anomalies by setting rules that define acceptable differences between the expected and actual values, either as an absolute value or a relative percentage. This helps reduce false anomalies caused by minor fluctuations, allowing you to focus on significant deviations.
+In the Feature selection pane, you can suppress anomalies by setting rules that define acceptable differences between the expected and actual values, either as an absolute value or a relative percentage. This helps reduce false anomalies caused by minor fluctuations, allowing you to focus on significant deviations.
```

Comment:
Please move the suppression rules outside of its own to feature section. that makes it easier to find.

Reply:
you also need to update example snapshot in my example.

Reply:
I removed this:

"Ensure that a feature, for example, logVolume, is properly defined in your model. Suppression rules are tied to specific features."

There is no photo snapshot currently in the docs but do you mean removing this:
```
Ignore anomalies for feature logVolume when the actual value is no more than 30% above the expected value.
Ignore anomalies for feature logVolume when the actual value is no more than 30% below the expected value.
```

and replace with screenshot of current UI?

Reply:
yeah, i think that would be more straightforward using screenshot.

Reply:
added 2 screenshots, still kept this wording though. 

First screenshot has entire feature configuration and second just a zoom in of the suppression rules, let me know if you think its too much

Reply:
In images/anomaly-detection/add-suppression-rules-absolute.png, you need use "Units" in both rules.

Reply:
fixed